packages/worker/src/worker-options.ts

```diff
@@ -407,11 +408,11 @@ export interface WorkerOptions {
   /**
    * A mapping of interceptor type to a list of factories or module paths.
    *
-   * By default, {@link ActivityInboundLogInterceptor} and {@link WorkflowInboundLogInterceptor} are installed. If you
-   * wish to customize the interceptors while keeping the defaults, use {@link appendDefaultInterceptors}.
-   *
    * When using {@link workflowBundle}, these Workflow interceptors (`WorkerInterceptors.workflowModules`) are not used.
    * Instead, provide them via {@link BundleOptions.workflowInterceptorModules} when calling {@link bundleWorkflowCode}.
+   *
+   * The SDK itself may automatically append interceptors required to support its own features Explicitely calling
+   * `appendDefaultInterceptors()` is no longer required.
```

Comment:
What happens if they are also provided, do we run them twice , or just ignore them...

Reply:
We would have run them twice. I modified code tonot prepend internal interceptors if they "seems" to already be there. But we can't catch _all_ cases that might results in logging wf/activity life cycle events twice.